src/Page/HelpSelfSingle.elm

```diff
@@ -200,9 +207,11 @@ categoryKeysFromSlug slug =
                   , linkHref = HelpSelfSeparatingResource2Href
                   }
                 ]
+        , image = Just "/Separating.svg"
         }
 
     else
         { title = HelpSelfCategoryNotFoundTitle
         , resources = Nothing
+        , image = Nothing
```

Comment:
If you have to do a maybe just to support not found page, I'd pick a random one (money related) and stick that in instead.